docs/deploying/kubernetes.html

```diff
@@ -102,31 +102,85 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
     </nav>
     <div id="doc-container" class="docs-item-4 docs-window">
       <main>
-        <h1 class="heading-1">Deploy to the Cloud with Kubernetes</h1>
-        <h2 class="heading-2">Prerequisites</h2>
-        <p>Whilst Kitura can be deployed to any Cloud offering we will be using IBM Cloud as an example in this guide.</p>
-        <ul class="plain-list">
-          <li><a onclick="setActiveSidebarElementForParent('create-server')" href="../getting-started/create-server.html#">Kitura Server:</a> Learn how to create one in our Getting Started guide.</li>
-          <li><span class="blue-text">IBM Cloud Account:</span> Learn how to sign up for free</li>
-          <li><span class="blue-text">A free cluster on IBM Cloud:</span> Learn how to create your free cluster</li>
-          <li><span class="blue-text">Deployed server to Docker:</span> Learn how to do this by following our Docker guide</li>
-        </ul>
+        <h1 class="heading-1">Deploying with Kubernetes</h1>
+        <p>Kubernetes is a platform which allows you to automate the deployment, scaling and management of your containerized applications.</p>
+        <p>You can use Kubernetes to deploy your Kitura application to any cloud provider, in this guide we will use IBM Cloud as our cloud provider.</p>
```

Comment:
You can deploy your Kitura application to any Kubernetes cluster. In this guide we will use the Kubernetes Service on IBM Cloud.

Reply:
@ianpartridge I've addressed your review comments.